dist/Locale-Maketext/Makefile.PL

```diff
@@ -0,0 +1,45 @@
+require 5.004;
```

Comment:
The module uses `our` and `no warnings;`, so this should be 5.006.

Reply:
Fixed, also made the INSTALLDIRS similar to the other examples you provided.